js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -948,7 +937,25 @@ const EditingController = modules.ViewController.inherit((function() {
         _beforeUpdateItems: function() { },
 
         _getVisibleEditRowIndex: function() {
-            return this._editRowIndex >= 0 ? this._editRowIndex - this._dataController.getRowIndexOffset() : -1;
+            const dataController = this._dataController;
+            const columns = this._columnsController.getColumns();
+
+            let editColumn;
+            columns.some(column => {
+                if(column.visibleIndex === this._editColumnIndex) {
```

Comment:
visibleIndex in column is not index of rendered column. Use `columnsController.getVisibleColumns()[this,_editColumnIndex]` to get column by rendered column index